src/App.tsx

```diff
@@ -18,13 +18,17 @@ import AutoUpdateDialog from './containers/AutoUpdateDialog/AutoUpdateDialog.tsx
 import useMining from '@app/hooks/mining/useMining.ts';
 
 import { LayoutGroup } from 'framer-motion';
+import { useLighting } from '@app/hooks/useLighting.ts';
+import { useUiMiningStateMachine } from './hooks/mining/useMiningUiStateMachine.ts';
 
 export default function App() {
     useAirdropSyncState();
     useSetUp();
     useMining();
     useGetStatus();
     useEnvironment();
+    useLighting();
```

Comment:
we don't want `useLighting` in anymore (we can actually remove that hook now :D)